### ISSUE TITLE
"published" value is null in REST API list results for /dois

### ISSUE THREAD

**Issue by KellyStathis** [datacite/datacite#2049]
#### Expected Behaviour

In the list results for /dois, `published` should have the value of publicationYear or Date with dateType "Issued". This should always be present because publicationYear is required in the metadata schema.
https://support.datacite.org/docs/what-is-the-difference-between-the-created-and-registered-date-in-the-datacite-rest-api

#### Current Behaviour
The value of `published` is null in some list results, although it is present in the singleton results.

#### Steps to Reproduce

Example list with results showing `"published": null`: https://api.datacite.org/dois?published=2024

![Screen Shot 2024-01-31 at 09 22 50](https://github.com/datacite/datacite/assets/17298577/25d75845-6e8e-4941-91c2-03b8086cf311)

Example single DOI from that list with "published": "2024": https://api.datacite.org/dois/10.15468/feuwzp

**Comment by codycooperross** [comment]
I believe this is a duplicate of datacite/lupo#850

**Comment by KellyStathis** [comment]
@codycooperross you're right, thanks! I had only searched this repository, not lupo. Which do you want to keep?
